docs/source/index.rst

```diff
@@ -47,7 +47,7 @@ Daml Documentation
    :hidden:
    :caption: Deploying to Daml ledgers
 
-   deploying
+   deploy/generic_ledger
```

Comment:
You probably need to reflect that change in the PDF one too.

Reply:
Thanks, added that.